### ISSUE TITLE
'overlap:false' resizing webview, conflicts with other plugins

### ISSUE THREAD

**Issue by rodrigograca31** [floatinghotpot/cordova-admob-pro#329]
As detailed here: [https://github.com/jcesarmobile/plusOneButtonPlugin/issues/2](https://github.com/jcesarmobile/plusOneButtonPlugin/issues/2)

**The problem:**
When `overlap` is not specified it assumes `false` as default... and with overlap set to `false` it makes the other plugin (+1 button) disappear when the ad loads....

Changing `overlap` to `true` solves the problem, but causes another problem: The ads overlap the app UI and I can't push the UI up since I don't know the ad height.... (`adSize: 'SMART_BANNER'`)

**Question:**
What causes the other plugin to disappear?

**Comment by floatinghotpot** [comment]
When overlap:false, the plugin will resize the cordova webview. If some other plugin also resize the webview, it might cause some conflict.

Now you try the overlap:true,  you will know the ad banner height by detecting pad/phone device and your app orientation. 

See: https://developers.google.com/admob/android/banner
![banner](https://cloud.githubusercontent.com/assets/2339512/13839970/4dd7dd9c-ec5a-11e5-8bb2-86af40e7c8c7.png)

**Comment by rodrigograca31** [comment]
Hi @floatinghotpot thanks for your answer.

The problem is that I'm using the `'SMART_BANNER'` that may have 32, 50 or 90....
Another issue is that 90px in the web view is not the same as 90 in the admob plugin....

I just want to be able to use `overlap: false` without making the other plugin disappear...

**Comment by VoDoV** [comment]
I'm having the same issue - other plugin views disappear when I enable the admob plugin with `overlap: false`.

Working with `overlap: true` is error-prone, so I stepped through the View setup and I found the cause: 
The `setContentView()` call in `GenericAdPlugin.showBanner`
**removes all other content views**. Fortunately there is a surprisingly easy fix: Replace the line
 
- `getActivity().setContentView(splitLayout);` with `parentView.addView(splitLayout, 0);` to preserve the content views.

- And remove the following `parentView.bringToFront();` to keep the other content views in front of the WebView.

As this part of the plugin is closed source I can't provide a push request, but I hope @floatinghotpot includes this patch in one of the next releases.
